switchtheme.module

```diff
@@ -187,6 +201,34 @@ function switchtheme_block_view($delta = '') {
   }
 }
 
+/**
+ * returns an unordered list of active themes to switch to.
+ */
+
+function switchtheme_return_itemlist_block() {
+  $themes = list_themes();
+  $enabledThemes = []; 
```

Comment:
Variables use this "camel case" only in JavaScript. In PHP we use underscores between "words".

Also please see https://api.backdropcms.org/php-standards#arrays

> PHP 5.4 introduced support for the shorthand format for arrays (see https://www.php.net/manual/en/migration54.new-features.php). Since Backdrop CMS supports PHP 5.3.2 (see [System Requirements](https://backdropcms.org/requirements)), the long format is to be used with arrays:

```suggestion
  $enabled_themes = array(); 
```